pkg/apply/applydrivers/apply_drivers_default.go

```diff
@@ -273,3 +268,30 @@ func (c *Applier) deleteCluster() error {
 	logger.Info("succeeded in deleting current cluster")
 	return nil
 }
+
+func (c *Applier) syncWorkdir() {
+	workDir := constants.ClusterDir(c.ClusterDesired.Name)
```

Comment:
if other node is not same user name ?

Reply:
yes !,it should be ~/. sealos/.

Reply:
> if exec sealos user is root and other node user name is not root , same this dir not using all nodes.
I'll make some modifications to this code